moncli/entities/item.py

```diff
@@ -338,17 +340,29 @@ def remove_files(self, file_column: en.cv.FileValue, *args):
                 url_thumbnail : `str`
                     Url to view the asset in thumbnail mode. Only available for images.
         """
+        value = json.dumps({} )
```

Comment:
This is not what I asked you to do earlier...

Just remove this line period.  It serves no purpose.